source/dub/dub.d

```diff
@@ -509,19 +509,20 @@ class Dub {
 				if (basename == rootbasename) continue;
 
 				if (!m_project.selections.hasSelectedVersion(basename)) {
-					logInfo("Non-selected package %s is available with version %s.",
-						basename, ver);
+					logInfo("Upgrade", Color.yellow,
+						"Non-selected package %s is available with version %s", basename, ver);
 					any = true;
 					continue;
 				}
 				auto sver = m_project.selections.getSelectedVersion(basename);
 				if (!sver.path.empty) continue;
 				if (ver.version_ <= sver.version_) continue;
-				logInfo("Package %s can be upgraded from %s to %s.",
-					basename, sver, ver);
+        logInfo("Upgrade", Color.yellow,
+          "%s can be upgraded %s -> %s",
+					basename.color(Mode.bold), sver, ver);
```

Comment:
you mixed in spaces to indent here

Reply:
In response to this and any other "mixed in spaces to indent" etc... 
I'm really sorry, I didn't check my editor before and I'm totally used to spaces to indent. My editor had 2 chars as tab width so I didn't spot any issue until I saw the code here on GitHub. I will fix everything.